runtime/test/fake-slot-manager.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+"use strict";
+
+let assert = require('chai').assert;
+
+class FakeSlotManager {
```

Comment:
On naming, this seems more of a MockSlotManager, since it contains expectations/assertions. I would expect a FakeSlotManager to be just a stub that does nothing.